package.json

```diff
@@ -2,10 +2,13 @@
   "name": "stipulate",
   "version": "0.0.5",
   "description": "A module extending the Fetch API with some useful default error handling and hooks.",
-  "main": "src/stipulate.js",
+  "main": "cjs/stipulate.js",
+  "module": "src/stipulate.js",
   "scripts": {
-    "test": "babel-node node_modules/.bin/isparta cover --root src/ --report text --report lcov --report html node_modules/.bin/_mocha --",
-    "coveralls": "cat coverage/lcov.info | node_modules/.bin/coveralls"
+    "compile-cjs": "scripts/compile-common-js.sh",
+    "coveralls": "cat coverage/lcov.info | node_modules/.bin/coveralls",
```

Comment:
IMO most of the scripts here can use improvement. That test script made my eye twitch.